daemon/api/main.go

```diff
@@ -3,3 +3,20 @@
 //go:generate oapi-codegen --config=codegen_client.yaml ./api.yaml
 
 package api
+
+import "fmt"
+
+func (t MonitorUpdateQueued) String() (out string) {
+	return fmt.Sprint(t.OrchestrationId)
+}
+
+func (t Problem) String() (out string) {
+	if t.Status != 200 {
```

Comment:
Relax this to  < 300 ?

Reply:
Better to display the code if not 200 to avoid confusion